v22.1/cockroachdb-feature-availability.md

```diff
@@ -1,27 +1,57 @@
 ---
-title: Features in Preview
-summary: Learn about the features available in preview in CockroachDB
+title: CockroachDB Feature Availability
+summary: Learn about the features available in preview and limited access in CockroachDB
 toc: true
 docs_area: reference.sql
 key: experimental-features.html
 ---
 
+CockroachDB features go through a development lifecycle and some are publicly available at different phases than others. This page defines the different levels of CockroachDB {{ page.version.version }} feature availability, and lists the features in each phase.
+
+## Feature availability phases
+
+Phase                                         | Definition | Accessibility 
+----------------------------------------------+------------+-------------
+Private preview                               | Feature is not production ready and will not be publicly documented. | Invite-only
+[Limited access](#features-in-limited-access) | Feature is production-ready but not available widely because of known limitations and/or because capabilities may change or be added based on feedback. | Opt-in </br>Contact your Cockroach Labs account team.
+[Preview](#features-in-preview)               | Feature is production-ready and publicly available. However, this feature may have known limitation and/or capabilities may change or be added based on feedback. | Public
+General availability (GA)                     | Feature is production-ready, publicly available, and has reached full development maturity. | Public
+
+## Features in limited access
+
+{{site.data.alerts.callout_info}}
+**The following features are in limited access** and are only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. These features are subject to change.
+{{site.data.alerts.end}}
+
+### Export logs from {{ site.data.products.dedicated }} clusters
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](../cockroachcloud/cloud-api.html) to configure [log export](../cockroachcloud/export-logs.html) to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, send log entries to specific log group targets by log channel, among others.
+
+## Features in preview
+
 This page lists the features that are available in preview in CockroachDB {{ page.version.version }}. These features are subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc/en-us).
 
-## Session variables
+### Export metrics from {{ site.data.products.dedicated }} clusters
```

Comment:
This is included twice. Also it's listed here as Preview but on the "metrics export"/export-metrics.html page, it's listed as Limited Access.